.devcontainer/features/languages/c/install.sh

```
@@ -0,0 +1,62 @@
+# Install C development packages
+# Support version parameter from devcontainer-feature.json (GCC version)
+GCC_VERSION="${VERSION:-latest}"
+echo -e "${YELLOW}Installing C compilers and development tools...${NC}"
+if [ "$GCC_VERSION" != "latest" ]; then
+    echo -e "${YELLOW}Requested GCC version: ${GCC_VERSION}${NC}"
+    GCC_PKG="gcc-${GCC_VERSION}"
+else
+    GCC_PKG="gcc"
+fi
+sudo apt-get update && sudo apt-get install -y \
+    ${GCC_PKG} \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)